docker/fuchsia/start.sh

```diff
@@ -13,6 +13,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+DEPLOYMENT_ZIP="linux-3.zip"
```

Comment:
I think we might need to rebuild the google images for this to be used.

Reply:
The plan is to use progressive rollout to validate on internal, we should not need to perform this operation for now.